07.label_semantic_roles/train.py

```diff
@@ -211,6 +214,7 @@ def event_handler(event):
         output_layer=predict,
         parameters=parameters,
         input=test_data,
+        feeding=feeding,
```

Comment:
The bug is here. The `feeding` should be set.

Reply:
I see.